automation/check-patch.e2e-ovs-cni-functests.sh

```diff
@@ -20,6 +20,10 @@ main() {
     source automation/check-patch.setup.sh
     cd ${TMP_PROJECT_PATH}
 
+    # until we fix OVS lane on the newer kubevirtci
```

Comment:
Mark this with TODO and link the tracker of the issue

Reply:
found the root cause it seems, will try to fix it from the root

Reply:
dropped since the root cause is fixed (need to merge ovs and kmp PRs for that)